msteams-platform/sbs-meeting-content-bubble.yml

```diff
@@ -67,40 +70,77 @@ items:
+
     [!INCLUDE [Azure client application](includes/get-started/azure-client-application.md)]
 
+    > [!div class="nextstepaction"]
+    > [I ran into an issue](https://github.com/MicrosoftDocs/msteams-docs/issues/new?template=Doc-Feedback.yaml&title=%5BI+ran+into+an+issue%5D+Add+client+application&author=surbhigupta&pageUrl=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fmicrosoftteams%2Fplatform%2Fsbs-meeting-content-bubble%3Ftabs%3Dngrok%252Csend-in-meeting-notification%252Cin-meeting-notification%26tutorial-step%3D3&contentSourceUrl=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fmsteams-docs%2Fblob%2Fmain%2Fmsteams-platform%2Fsbs-meeting-content-bubble.yml%23L60&documentVersionIndependentId=f1b9491a-07c9-c915-6733-58eb905ec502&metadata=%2A%2BID%253A%2Be473e1f3-69f5-bcfa-bcab-54b098b59c80%2B%250A%2A%2BService%253A%2B%2Amsteams)
+
 - title: Create your bot 
   durationInMinutes: 5
   content: |
 
     [!INCLUDE [Azure bot resource](includes/get-started/azure-bot-resource.md)]    
 
+    > [!div class="nextstepaction"]
+    > [I ran into an issue](https://github.com/MicrosoftDocs/msteams-docs/issues/new?template=Doc-Feedback.yaml&title=%5BI+ran+into+an+issue%5D+Create+an+Azure+bot+resource&author=surbhigupta&pageUrl=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fmicrosoftteams%2Fplatform%2Fsbs-meeting-content-bubble%3Ftabs%3Ddev%252Csend-in-meeting-notification%252Cin-meeting-notification%26tutorial-step%3D4&contentSourceUrl=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fmsteams-docs%2Fblob%2Fmain%2Fmsteams-platform%2Fsbs-meeting-content-bubble.yml%23L124&documentVersionIndependentId=f1b9491a-07c9-c915-6733-58eb905ec502&metadata=%2A%2BID%253A%2Be473e1f3-69f5-bcfa-bcab-54b098b59c80%2B%250A%2A%2BService%253A%2B%2Amsteams)
+
     [!INCLUDE [Teams channel](includes/get-started/add-teams-channel.md)]
 
+    > [!div class="nextstepaction"]
+    > [I ran into an issue](https://github.com/MicrosoftDocs/msteams-docs/issues/new?template=Doc-Feedback.yaml&title=%5BI+ran+into+an+issue%5D+Add+a+Teams+channel&author=surbhigupta&pageUrl=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fmicrosoftteams%2Fplatform%2Fsbs-meeting-content-bubble%3Ftabs%3Ddev%252Csend-in-meeting-notification%252Cin-meeting-notification%26tutorial-step%3D4&contentSourceUrl=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fmsteams-docs%2Fblob%2Fmain%2Fmsteams-platform%2Fsbs-meeting-content-bubble.yml%23L124&documentVersionIndependentId=f1b9491a-07c9-c915-6733-58eb905ec502&metadata=%2A%2BID%253A%2Be473e1f3-69f5-bcfa-bcab-54b098b59c80%2B%250A%2A%2BService%253A%2B%2Amsteams)
```

Comment:
[I ran into an issue] Add Teams channel to an Azure bot resource

Reply:
https://github.com/MicrosoftDocs/msteams-docs/blob/main/msteams-platform/sbs-meeting-content-bubble.yml#L100